packages/back-end/src/integrations/Athena.ts

```diff
@@ -45,4 +54,52 @@ export default class Athena extends SqlIntegration {
   ensureFloat(col: string): string {
     return `1.0*${col}`;
   }
+  async getInformationSchema(): Promise<InformationSchema[]> {
+    const sql = `SELECT
+        table_name,
+        table_catalog,
+        table_schema,
+        count(column_name) as column_count
+      FROM
+        information_schema.columns
```

Comment:
Athena is basically just a hosted version of PrestoDB.  In Presto, we're using the default catalog here and for Athena, we do have a `catalog` setting as well.

Reply:
Updated!